packages/atomic/src/components/atomic-pager-v1/atomic-pager.pcss

```diff
@@ -0,0 +1 @@
+@import '../atomic-results-per-page-v1/atomic-results-per-page.pcss';
```

Comment:
I would prefer to avoid a component inheriting the style of another one. Instead, could this styling be extracted into a more generic file, which both components could import?